config/locales/en/shared.yml

```diff
@@ -165,36 +234,54 @@ en:
       press_release:
         one: Press release
         other: Press releases
+      procurement:
+        one: Procurement
+        other: Procurement
       promotional:
         one: Promotional material
         other: Promotional material
       publication:
         one: Publication
         other: Publications
+      publication_scheme:
+        one: Publication scheme
+        other: Publication schemes
       raib_report:
         one: Rail Accident Investigation Branch report
         other: Rail Accident Investigation Branch reports
+      recruitment:
+        one: Recruitment
+        other: Recruitment
       regulation:
         one: Regulation
         other: Regulations
       research:
-        one: Research and analysis
```

Comment:
Are we sure about this content change? I'd suggest leaving it unchanged for the moment.

Reply:
Heya, have changed this so it stores the existing Collections translations rather than overwriting them :) excellent spot!